jung-algorithms/src/main/java/edu/uci/ics/jung/layout/algorithms/FRBHLayoutAlgorithm.java

```diff
@@ -70,6 +71,8 @@
 
   private BarnesHutQuadTree<N> tree;
 
+  private Random random = new Random();
```

Comment:
This should give the same values for random.nextDouble() as we got from Math.random().
Using Random allows a settable seed value for repeatable tests

Reply:
yeah, we should be doing something like this everywhere we have random numbers, thanks :)